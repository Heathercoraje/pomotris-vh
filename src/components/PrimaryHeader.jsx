import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';
import orange from '../../img/orange.png';

class PrimaryHeader extends Component {
	render() {
		return (
			<header className="primary-header">
				<div>
					<span className="logo">
						<Link to="/">
							Pomotris<img className="logo-icon" src={orange} />
						</Link>
					</span>
				</div>
				<ul>
					<li>
						<Link to="/howto">How to</Link>
					</li>
					<li>
						<Link to="/feedback">Feedback</Link>
					</li>
				</ul>
			</header>
		);
	}
}

export default PrimaryHeader;
