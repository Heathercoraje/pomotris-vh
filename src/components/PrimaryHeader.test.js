import React from 'react';
import { shallow } from 'enzyme';
import PrimaryHeader from './PrimaryHeader';

describe('setup', () => {
	it('passes', () => {
		expect(true).toBe(true);
	});
});
