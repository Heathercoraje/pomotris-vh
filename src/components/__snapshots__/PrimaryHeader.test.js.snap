// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryHeader has links 1`] = `
<header
  className="primary-header"
>
  <div>
    <span
      className="logo"
    >
      <Link
        replace={false}
        to="/"
      >
        Pomotris
        <img
          className="logo-icon"
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </Link>
    </span>
  </div>
  <ul>
    <li
      className="menu-item"
    >
      <Link
        replace={false}
        to="/about"
      >
        About
      </Link>
    </li>
  </ul>
</header>
`;
